python/semantic_kernel/connectors/ai/open_ai/services/open_ai_handler.py

```diff
@@ -51,6 +51,7 @@ class OpenAIHandler(KernelBaseModel, ABC):
     prompt_tokens: int = 0
     completion_tokens: int = 0
     total_tokens: int = 0
+    instruction_role: str = Field(default="system")
```

Comment:
if we do 
```suggestion
    instruction_role: Literal['system', 'developer'] = Field(default="system")
```
we get auto completion!

Reply:
I thought about that, but that now constrains up to have to make an update to support other strings. It's less maintenance for us to leave it as free text.

Reply:
Also, I just realized this `instruction_role` doesn't even need to exist here in the OpenAIHandler as it exists in `ChatCompletionClientBase`. Given Pydantic routes the attribute properly, I don't want to constrain the instruction_role at the base level with any particular value.